src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -280,7 +280,7 @@ private void ProcessPaths(List<string> resolvedPaths)
         /// </summary>
         [Parameter(ParameterSetName = FromSourceParameterSetName)]
         [Parameter(ParameterSetName = FromMemberParameterSetName)]
-        public Language Language { get; set; } = Language.CSharp;
+        public Language Language { get; set; }
```

Comment:
I suspect some of the enum initialisation is preferred, since it's more explicit about intent. I will ask for more reviewers

Reply:
I agree. My comment above is about the same.